.github/workflows/publish-pg_lakehouse.yml

```diff
@@ -0,0 +1,134 @@
+  push:
+    tags:
+      - "v*"
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The version to set for the pg_lakehouse release. This publishes the latest commit of the chosen branch and uploads it to the pre-existing GitHub Release of the provided version."
+        required: true
+        default: ""
+
+concurrency:
+  group: publish-pg_lakehouse-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish-pg_lakehouse:
+    name: Publish pg_lakehouse for PostgreSQL ${{ matrix.pg_version }} on ${{ matrix.arch }}
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            pg_version: 15
+            arch: amd64
+          - runner: ubicloud-standard-8-ubuntu-2204-arm
+            pg_version: 15
+            arch: arm64
+          - runner: ubuntu-latest
+            pg_version: 16
+            arch: amd64
+          - runner: ubicloud-standard-8-ubuntu-2204-arm
+            pg_version: 16
+            arch: arm64
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Retrieve Ubuntu & GitHub Tag Versions
+        id: version
+        run: |
+          # If no workflow_dispatch version is provided, we use workflow tag trigger version
+          if [ -z "${{ github.event.inputs.version }}" ]; then
+            echo "version=${GITHUB_REF#refs/tags/v}" >> $GITHUB_OUTPUT
+          else
+            echo "version=${{ github.event.inputs.version }}" >> $GITHUB_OUTPUT
+          fi
+          echo "ubuntu_version=$(lsb_release -rs | sed 's/\.//')" >> $GITHUB_OUTPUT
+
+      - name: Install & Configure Supported PostgreSQL Version
+        run: |
+          wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          sudo apt-get update && sudo apt-get install -y postgresql-${{ matrix.pg_version }} postgresql-server-dev-${{ matrix.pg_version }} debhelper devscripts dput gnupg
+          sudo chown -R $(whoami) /usr/share/postgresql/${{ matrix.pg_version }}/extension/ /usr/lib/postgresql/${{ matrix.pg_version }}/lib/
+          echo "/usr/lib/postgresql/${{ matrix.pg_version }}/bin" >> $GITHUB_PATH
+
+      # Note: We require a different version of pgrx (0.11.3) due to version compatibility
+      # with the supabase/wrappers project on which pg_lakehouse is based
+      - name: Install pgrx
+        run: cargo install -j $(nproc) --locked cargo-pgrx --version 0.11.3
+
+      - name: Initialize pgrx for Current PostgreSQL Version
+        working-directory: pg_lakehouse/
+        run: cargo pgrx init --pg${{ matrix.pg_version }}=/usr/lib/postgresql/${{ matrix.pg_version }}/bin/pg_config
+
+      # TODO: Add telemetry to pg_lakehouse
```

Comment:
@rebasedming could you quickly connect the telemetry to `pg_lakehouse`, similar to how we do it in `pg_analytics/search`? It should be as simple as invoking the background worker

Reply:
Sadly there's no straightfoward way because it requires `shared`, but `shared` and `pg_lakehouse` are on different pgrx versions.

Reply:
Ah shoot, this is not good. I really think we should ship this with telemetry